src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -169,10 +169,13 @@ private void addAliasToAllProjects(final OpenshiftRequestContext context, List<S
             if (filteredProjects.isEmpty()) {
                 filteredProjects.add(BLANK_PROJECT);
             } else {
-                String alias = UserRolesSyncStrategy.formatAllAlias(context.getUser());
-                LOGGER.debug("Adding indexes to alias '{}' for user '{}'", alias, context.getUser());
-                buildUserAlias(filteredProjects, projectPrefix, alias);
-                filteredProjects.add(alias);
+                // only add alias if kibana index mode is UNIQUE
+                if(context.getKibanaIndexMode() == UNIQUE) {
```

Comment:
This is not quite correct.  An alias is added to simplify creating searches and visualizations across multiple indices.  I would expect maybe you would want a 'shared all' alias for non-ops users similar to the '.all' alias for ops

Reply:
This was a fix for the username alias, because it was shared among all users and it was a bit weird.

Reply:
We are unable to use '.all' since alias names are global to the ES cluster.  We would just need to settle on something like '.all_non_ops' or something of the like.  I still am curious how this interplay will work given that users sharing the same kibana index will likely not share the same project indices.  Can you articulate a usecase or general motivation for this change?

Reply:
The usecase or the request from our customers are currently the same when we rollout the EFK. They want to share searches and dashboards with each other, this is needed because our customers have a dedicated OCP for their users. I did not think about an alias to be honest and don't see currently the need, because as you mentioned, we maybe face the situation when not all users are share the same projects.

Reply:
@portante What kind of issues due you think you will see with visualizations and dashboards if they are built using indices that may be only visible to some users?  Does this mean you will see the error bar at the top of Kibana for certain UI objects?

Reply:
@jcantrill our QA team is currently testing the changes, will let you know if there are any findings.
If there is a saved search for example from for an index where the user don't have access then a security exception will be shown.
<img width="1262" alt="access_index_without_permission" src="https://user-images.githubusercontent.com/2911849/29818094-031f1efc-8cbb-11e7-8a07-19aed125b9d8.png">

Reply:
If somebody decides to use the `shared_non_ops` mode in a multi-tenant environment, does that even make sense?  But if somebody deploys their environment, where all users can see all non-operations related projects, then would be nice for sharing views.

Reply:
@portante that's right, this option is not intended to be used on a multi-tenant environment. Our customer environments are configured in the way that users are seeing all non-operations related projects.